### ISSUE TITLE
Multiple domains - global Administrator and Server Rights

### ISSUE THREAD

**Issue by vitko-bg** [Ylianst/MeshCentral#974]
Finally I managed to setup multi domains on config.json and started testing it. After a couple of tests I think there are some "problems". So in my case I do have a MAIN Domain and SUB domains let's say:
 "domains": {
    "": {                   --> I call this one the MAIN
        ......
    },
    "subdomain1":{         --> SUB Domain 1
        ......
    },
    "subdomain2":{         --> SUB Domain  2 and so on.
        ......
    }
I created couple of users in MAIN and SUB Domains. First of all it is great that is working that way - some of remote offices will be able to look after them selves on their own :)
But I still cannot figure out the next problems:
 - what needs to be done to make possible for Administrators in SUB Domains NOT to have access to My Server Tab ( most important! for me as I do not trust anyone :) ) and still to remain as a Administrator within the SUB Domain?
 - How Administrator from MAIN domain can see all groups, users, devices not only in the MAIN Domain, but also within all SUB Domains?
In short is there a way to have a GLOBAL Administrator - which can do, see and access everything (users, usergroups, devices, device groups, server rights, etc) including all info within SUB domains) and a Local Administrators (on SUB Domain level), with no Server rights and not being able to see and access GLOBAL Administrator.
Thanks.

**Comment by Ylianst** [comment]
Being able to disable the "My Server" tab on any domain is an excellent ask. I will be adding this. There is currently no global administrator, domains are like separate servers right now. However, I am thinking of working on some global administrator that could edit the config.json / reset the server, etc. This would be especially useful for instances running in Docker.

I am back from vacation tomorrow and will start working on improvements and feature again.

**Comment by vitko-bg** [comment]
> Being able to disable the "My Server" tab on any domain is an excellent ask

Yes - I think it will be right just a MAIN Administrator to have access to the server tab, instead of all other Administrators.
In regards to GLOBAL Administrator - I believe it is not easy task - but what about MAIN Administrator(from main domain) to have access to all other Domains. I mean username and password for Main Domain Administrator to work on domain1.meshcentral.com, domain2....... and so on? And not to be visible in the subdomains Users tab ...... Is that possible? Thanks.

**Comment by vitko-bg** [comment]
Hi Ylianst - is there any update on this request? I know that you are very busy with more important BUGs instead of Enhancements. Will really appreciate if this can be done.
Thanks.

**Comment by MordyT** [comment]
Related: https://github.com/Ylianst/MeshCentral/issues/724

**Comment by vitko-bg** [comment]
Is there any update on the above request? I know that this is an extra job in addition to current working version, but it is really important for me and infrastructure within the company. Cannot trust other administrators on subdomains ...... Thanks in advance....
